sycl/test-e2e/Basic/buffer/buffer_create.cpp

```diff
@@ -1,7 +1,7 @@
 // REQUIRES: gpu,level_zero
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env ZE_DEBUG=1 %GPU_RUN_PLACEHOLDER %t.out 2> %t1.out; cat %t1.out %GPU_CHECK_PLACEHOLDER
-// UNSUPPORTED: ze_debug-1,ze_debug4
+// UNSUPPORTED: ze_debug-1,ze_debug4,ze_debug6
```

Comment:
I think it is overkill to have the test harness know about various possible values of ZE_DEBUG
Can we instead just have single "ze_debug" one to mean any non-0 setting of ZE_DEBUG?
Or teach the harness to know it's a bit-mask, and then only "ze_debug4" is enough to filter all -1, 4, and 6 values.